src/main/resources/io/viash/platforms/nextflow/WorkflowHelper.nf

```diff
@@ -603,10 +606,467 @@ def paramsToList(params, config) {
 }
 
 def paramsToChannel(params, config) {
+  System.err.println("Warning: paramsToChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   Channel.fromList(paramsToList(params, config))
 }
 
 def viashChannel(params, config) {
+  System.err.println("Warning: viashChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   paramsToChannel(params, config)
     | map{tup -> [tup.id, tup]}
 }
+
+/**
+ * From a viash config, gather the config entries 
+ * for arguments that have property 'multiple: true'
+ */
+
+private List<Map> _getMultiParameterSettings(Map config) {
```

Comment:
Can this be removed? It seems like it's not being used.

If it is, I think `config.functionality.allArguments.findAll{it.multiple}` is more readable than `_getMultiParameterSettings(config)`, so I'd still propose removing it.

Reply:
Done!